modules/ROOT/pages/query-unified-schema.adoc

```diff
@@ -5,7 +5,7 @@ The unified schema resolves into a single GraphQL endpoint. To run queries or mu
 == Before You Begin
 
 * You must have xref:permissions.adoc[the Consume, Contribute, or Admin permission]. 
-* Review the xref:index.adoc#restrictions-and-limitations[restrictions and limitations] for REST API calls and queries.
+* Review xref:index.adoc#anypoint-datagraph-api-support[Anypoint DataGraph API Support]
```

Comment:
I think a "the" is missing. It should read "Review _the_ Anypoint DataGraph API Support"